extensions/adobe/experience/workfront/changeevent.schema.json

```diff
@@ -80,6 +80,24 @@
           "title": "Created By ID",
           "type": "string",
           "description": "The employeeID involved in the CREATE event"
+        },
+        "workfront:RecFull": {
```

Comment:
we generally have property name start with a lower case.
PLease change it to  "workfront:recFull"